libchisel/src/trimexports.rs

```diff
@@ -40,6 +33,15 @@ impl ExportWhitelist {
             _ => Err(()),
         }
     }
+}
+
+impl ExportWhitelist {
```

Comment:
Why move this?

Reply:
Out of habit I put trait impls before method impls.